ci/main.sh

```diff
@@ -25,19 +25,30 @@ cmakeopts=(
 [ "$BUILD_MODE" = "coverage" ] && cmakeopts+=("-DENABLE_COVERAGE=yes")
 [ "$BUILD_MODE" = "sanitize" ] && cmakeopts+=("-DENABLE_SANITIZERS=yes")
 
+if [[ "$(get_os)" = "msys" ]]; then
+  cmakeopts+=("-G")
+  cmakeopts+=("MSYS Makefiles")
```

Comment:
I think you can do this in one line: `cmakeopts+=("-G" "MSYS Makefiles")`

Reply:
@ni4 feel free to fix this and merge anytime. Thanks!

Reply:
@ronaldtse done!